src/components/ContentListTable/components/EmptyTableState.tsx

```diff
@@ -32,7 +35,7 @@ const EmptyTableState = ({ notFiltered, clearFilters }: Props) => {
           ? 'No custom repositories'
           : 'No custom repositories match the filter criteria'}
       </Title>
-      <EmptyStateBody>
+      <EmptyStateBody className={classes.emptStateBody}>
```

Comment:
This fixes a padding issue I noticed with the button and the accompanying text above it.